developer/reference/components/root-style-registry.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: "<RootStyleRegistry>"
+description: The `<RootStyleRegistry>` component provides support for Makeswift's CSS-in-JS runtime in Next.js' App Router.
+---
+
+import AppProviders from "/snippets/app-router/providers.mdx";
+import AppLayout from "/snippets/app-router/layout.mdx";
+import PagesApp from "/snippets/pages-router/pages-app.mdx";
+
+## Props
+
+
+<ParamField query="cacheKey" type="string" default="mswft">
+  Optional key used when creating a Emotion cache for styles. Pass a unique value to avoid conflicts when multiple Emotion instances share the same key.
```

Comment:
```suggestion
  Optional key used when creating an [Emotion cache](https://emotion.sh/docs/@emotion/cache) for styles. Pass a unique value to avoid conflicts when multiple Emotion instances share the same key.
```

Reply:
Is that the correct link for the emotion cache?